cmake/wsrep.cmake

```diff
@@ -43,10 +35,9 @@ EXECUTE_PROCESS(
 #FILE(WRITE "wsrep_config" "Debug: WSREP_API_VERSION result: ${RESULT}\n")
 STRING(REGEX REPLACE "(\r?\n)+$" "" WSREP_API_VERSION "${WSREP_API_VERSION}")
 
-SET(WSREP_PACKAGE_VERSION "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}")
-SET(WSREP_VERSION
-    "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}.r${WSREP_PATCH_REVNO}"
-)
+SET(WSREP_VERSION "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}"
```

Comment:
> And I think a closing parenthesis is missing on line 38.

Closing parenthesis matching line 38 can be found either from line 42 or from 39 depending on which line numbers you mean.